lib/mosaic-objects/src/main/java/org/eclipse/mosaic/lib/objects/DuplicatableMessage.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.lib.objects;
+
+import org.eclipse.mosaic.lib.objects.v2x.MessageRouting;
+
+/**
+ * Creates a copy of the message
+ * @param <MessageT>
+ */
+public interface DuplicatableMessage<T extends DuplicatableMessage<T>> {
```

Comment:
Merriam-Webster says that the  more common adjective of duplicate would be *duplicable.* 🤷

Reply:
The word _duplicable_ is more common but _duplicatable_ is used to emphasize the process of making a copy as in "The data is easily duplicatable using this software." versus  "These documents are duplicable with the right equipment."

I would suggest using the word _duplicatable_ here but I don't mind using _duplicable_ if to you it sounds better :)